<launch>
    
    <!-- Scan Context -->
    <param name="sc_dist_thres" type="double" value="0.3"/> <!-- SC-A-LOAM, if want no outliers, use 0.1-0.15 -->
    <!-- <param name="sc_max_radius" type="double" value="20.0"/> 20 or 40 for indoor -->
    <param name="sc_max_radius" type="double" value="80.0"/> <!-- for outdoor -->

    <!-- utils -->
    <param name="save_directory" type="string" value="/EXTERNAL/homes/mengshj/new_catkin_ws/src/LCD/save_data/"/>  <!-- CHANGE THIS and end with / -->

    <!--  -->
    <param name="lidar_type" type="string" value="HDL64"/> 
     <remap from="/velodyne_points" to="/scan"/><!--/points_raw -->

    <!-- <node pkg="aloam_velodyne" type="ascanRegistration" name="ascanRegistration" output="screen" />
    <node pkg="aloam_velodyne" type="alaserOdometry" name="alaserOdometry" output="screen" />
    <node pkg="aloam_velodyne" type="alaserMapping" name="alaserMapping" output="screen" /> -->
    <!-- SC-A-LOAM -->
    <node pkg="LCD" type="lcd" name="lcd" output="screen" /> 


</launch>
